processing/src/main/java/io/druid/query/search/SearchQueryQueryToolChest.java

```diff
@@ -193,7 +208,9 @@ public SearchQueryQueryToolChest(
           @Override
           public Object apply(Result<SearchResultValue> input)
           {
-            return Lists.newArrayList(input.getTimestamp().getMillis(), input.getValue());
+            return dimensionSpecs != null
```

Comment:
will dimensionSpecs ever be null? Looks like it will either be an empty list or the query's dimension spec list